kiwi/system/setup.py

```diff
@@ -115,7 +115,7 @@ def import_repositories_marked_as_imageinclude(self):
         imageinclude attribute should be permanently added to
         the image repository configuration
         """
-        repository_sections = self.xml_state.get_repository_sections()
+        repository_sections = self.xml_state.get_image_repository_sections()
```

Comment:
Yes, that looks great to me, I am not very good at naming stuff... I'll change that, I'm totally fine with more explicit names.